### PR TITLE
Improve utility options

### PR DESCRIPTION
**Pull request by stanford-scs**
Added a --base64 option that modifies --printtxn and --signtxn.
Also added a --sec2pub option to produce the public key corresponding to a private key.

### CODE REVIEW

File: docs/software/commands.md [hunk 1]
```diff
@@ -48,6 +48,12 @@ forcescp doesn't change the requirements for quorum so although this node will e
   Note that if you do not have a STELLAR_NETWORK_ID environment
   variable, then before this argument you must specify the --netid
   option.
+* **--base64**: When preceding --printtxn or --signtxn, alters the
+  behavior of the option to works on base64-encoded XDR rather than
```
Comment (vogel) [inline docs/software/commands.md]: probably it should say "to work"

File: src/main/dumpxdr.cpp [hunk 2]
```diff
@@ -96,8 +97,10 @@ dumpxdr(std::string const& filename)
                                  xdr_strerror(errno));                         \
     } while (0)
 
+static std::string readFile(const std::string& filename, bool base64 = false);
```
Comment (vogel) [inline src/main/dumpxdr.cpp]: is this needed here?

Comment (stanford-scs) [inline src/main/dumpxdr.cpp]: It's just for the default argument.  But I don't feel strongly about it for a function that is only called in a few places.

Comment (stanford-scs) [inline src/main/dumpxdr.cpp]: Anyway, just got rid of it.

**Comment by vogel** [comment]
r+ d75ef349fa2e76e685a98f9d0dd34b7fb7df907c

**Pull request merged**